docs/website/02_examples/computePi.md

```diff
@@ -4,9 +4,132 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 -->
 
 # Compute the value of $\pi$
-The examples `iks001_computePi.cpp` shows the calculation of $\pi$ by computing the area
+
+## Description
+
+The example `iks001_computePi.cpp` shows the calculation of $\pi$ by computing the area
 and circumference of a unit circle. This example helps to understand the `Grid` module from Dune and the refinement techniques it
 brings. The example shows that a global refinement doesn't refine the number of grid entities on the boundary
 of the circle, which leads to a poor approximation of $\pi$ while it comparing with the area and circumference of the circle.
 On the other hand, it also shows how elements on the boundaries can be marked and refined, thereby resulting in an
-accurate approximation of $\pi$.
\ No newline at end of file
+accurate approximation of $\pi$.
+
+## Code highlights
+
+This example contains the following two functions
+```cpp
+boundaryUnawareRefinedCircle();
+boundaryAwareRefinedCircle();
+```
+demonstrating the usage of certain attributes of the grid module from Dune[@sander2020dune].
+Firstly, we explain the function `#!cpp boundaryUnawareRefinedCircle();`.
+
+Here, a `circleCoarse.msh`, which was created using [Gmsh](https://gmsh.info/), is read using the `Dune::GmshReader` and a `Dune::ALUGrid` object is created as shown below:
+```cpp
+constexpr int gridDim = 2;  // (1)
+using Grid            = Dune::ALUGrid<gridDim, 2, Dune::simplex, Dune::conforming>;
+auto grid             = Dune::GmshReader<Grid>::read("auxiliaryFiles/circleCoarse.msh", false);
+auto gridView         = grid->leafGridView();  // (2)
+```
+It is to note that dune-grid only supports Version 2 of the Gmsh format. The `#!cpp draw(gridView);` functionality is 
+included within the Ikarus framework to quickly draw grids and verify it for any major errors.
+The function `#!cpp grid->globalRefine(1);` is invoked to refine the grid on a global level. This means that, if we have 
+a single square-shaped 4-node quadrilateral element, the `globalRefine(1)` function will bisect the element once in 
+either direction and thereby result in a grid with 2 elements in either direction. This type of refinement is done in 
+the following and the area of the circle is compared to the value of $\pi$. The area of the circle itself is obtained by 
```

Comment:
It reads like we are using quad elements here. But you olny refer to the type of refinmenent maybe we you can make it clearer by "This type of refinement is done for the triangular elements"?